### ISSUE TITLE
Readme.md update for Windows/powershell users

### ISSUE THREAD

**Issue by safeerraees** [IHTSDO/snowstorm#209]
For windows, when you are trying to start the server up, you have to put the path to the jar file in quotes otherwise it doesn't work.
Just a minor issue, but will help other people like me save some time trying to figure out why it is not working.

**Comment by pgwilliams** [comment]
Hi @safeerraees, thanks for that.   Do you have a space anywhere in your jar's path?   Thanks!

**Comment by safeerraees** [comment]
No, I don't have any space in my file path. Also, one more thing I didn't mention in the above issue is that _snowstorm*.jar_ doesn't work you have to type the exact filename, which is I think is because of the quotes.
